### DISCUSSION TITLE
Strange navigation issues

### DISCUSSION THREAD

**Discussion by manuelmeurer** [nolimits4web/swiper#7259] [Q&A]
I have added a swiper to our website with the following config:

```
slidesPerView: "auto"
spaceBetween:  30
loop:          true
keyboard:      true
autoplay:      true
```

Everything works fine so far, but the navigation acts very strangely, and I can't figure out why. I tried adding and removing various options, to no avail...

You can see the live swiper here: https://uplink.tech/#profiles

Basically it doesn't loop, if you click on the right arrow below the slides, it stops at some point. And if you click on the left arrow, it jumps an arbitrary number of slides to the left (sometimes 2, sometimes 3 or 4).

Thankful for any tips on how to fix this! 😀

**Comment by adrianahdez** [comment]
In case you still have the issue, I had the same problem or similar (with version 11.1.1), and in my case, it was because I had some SCSS rules that were breaking the Swiper Javascript, though I don't know why. And I must say that those rules had nothing to do with the slider; they weren't directly applying to any slider elements. 
The problem disappeared only when I deactivated the loop, but came again when I activated it.
It doesn't make sense, I know, but I can confirm that SCSS rules was the cause in my situation. So, to debug the issue, I recommend to deactivate any SASS you have in the project. I hope this helps you solve it :)

**Comment by manuelmeurer** [comment]
Thanks @adrianahdez! I could eventually fix the problem and apparently forgot to post the solution here... 🙈
Unfortunately I forgot what the solution was. 😅 I THINK the issue was fixed in an update to Swiper that came out shortly afterwards...
I'll close this for now, feel free to reopen if the issue still occurs with the current version.
